### PR TITLE
feat: implement soundboard features

### PR DESCRIPTION
**Pull request by Dasupergrasskakjd**
## Summary

Soundboard(This is not finished and still has a lot to work on), just so we could work on it together

## Information

- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [x] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [ ] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.
- [ ] I have updated the changelog to include these changes.

### CODE REVIEW

**Review by Lulalaby — COMMENTED** [review]
Nice work so far!

**Comment by Dasupergrasskakjd** [comment]
I'm not sure what to work on next since I went on vacation for a few weeks and I'm completely lost when I came back, could anyone help me?

File: discord/state.py [hunk 1]
```diff
@@ -1843,6 +1846,47 @@ def create_message(
     ) -> Message:
         return Message(state=self, channel=channel, data=data)
 
+    def parse_voice_channel_effect_send(self, data) -> None:
+        json = __import__("json")
+        print(json.dumps(data, indent=4))
```
Comment (Dorukyum) [inline discord/state.py:1851]:
```suggestion
```

**Comment by Lulalaby** [comment]
@Dasupergrasskakjd any reason you closed this?

**Pull request closed without merging**